extra/zabbix/vpoller-module/vpoller.c

```diff
@@ -327,7 +327,17 @@ zbx_module_vpoller(AGENT_REQUEST *request, AGENT_RESULT *result)
 
   zmq_msg_close(&msg_in);
   zmq_close(zsocket);
-  
+
+  if (strcmp(result, "Task host.get failed") == 0) {
```

Comment:
There are other possible error messages, each for each corresponding task of vPoller.

I don't have enough time to look into this one now, but we should probably return a more generic error message to clients, so that we can match it here.

Reply:
Task * failed is only thrown in py-vpoller/src/vpoller/task/decorators.py
So this one just fixes if the VCSA or the standalone host isn't reached.

I don't think more generic is better - i feel it's worse. What if we define an keyword (ERROR, FAIL) and search for this in the zbx module and then just presenting it? I would contribute this.

But would be great if you have a look in this one first if you find time.